---
title: 在 "Try" 语句中，"Catch" 不能出现在 "Finally" 之后
ms.date: 07/20/2015
f1_keywords:
- vbc30379
- bc30379
helpviewer_keywords:
- BC30379
ms.assetid: 33d1278b-cf10-4c66-aaf8-08a4372f370b
ms.openlocfilehash: 0c0a24190190af28a761c318c3edc5f731424730
ms.sourcegitcommit: 0888d7b24f475c346a3f444de8d83ec1ca7cd234
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 12/22/2018
ms.locfileid: "53761475"
---
# <a name="catch-cannot-appear-after-finally-within-a-try-statement"></a><span data-ttu-id="4b80e-102">在 "Try" 语句中，"Catch" 不能出现在 "Finally" 之后</span><span class="sxs-lookup"><span data-stu-id="4b80e-102">'Catch' cannot appear after 'Finally' within a 'Try' statement</span></span>
<span data-ttu-id="4b80e-103">`Catch` 语句出现在代码中结束 `Try` 语句块的 `Finally` 之后。</span><span class="sxs-lookup"><span data-stu-id="4b80e-103">A `Catch` statement appears in the code after the `Finally` ending a `Try` statement block.</span></span> <span data-ttu-id="4b80e-104">`Catch` 必须出现在 `Try...Catch...Finally` 语句块之内。</span><span class="sxs-lookup"><span data-stu-id="4b80e-104">`Catch` must appear within a `Try...Catch...Finally` statement block.</span></span>  
  
 <span data-ttu-id="4b80e-105">**错误 ID:** BC30379</span><span class="sxs-lookup"><span data-stu-id="4b80e-105">**Error ID:** BC30379</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="4b80e-106">更正此错误</span><span class="sxs-lookup"><span data-stu-id="4b80e-106">To correct this error</span></span>  
  
1.  <span data-ttu-id="4b80e-107">将 `Catch` 语句移到代码中更合适的位置。</span><span class="sxs-lookup"><span data-stu-id="4b80e-107">Move the `Catch` statement to a more appropriate place in the code.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="4b80e-108">请参阅</span><span class="sxs-lookup"><span data-stu-id="4b80e-108">See Also</span></span>  
 [<span data-ttu-id="4b80e-109">Try...Catch...Finally 语句</span><span class="sxs-lookup"><span data-stu-id="4b80e-109">Try...Catch...Finally Statement</span></span>](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
